src/components/FbxLogo/FbxLogo.stories.js

```diff
@@ -0,0 +1,56 @@
+import { storiesOf } from '@storybook/vue'
+import { withInfo } from 'storybook-addon-vue-info'
+
+import FbxLogo from './FbxLogo.vue'
+import summary from './FbxLogo.md'
+
+const stories = storiesOf('Components/FundboxLogo', module)
```

Comment:
I think this should just be called `Logo`, since in the Storybook sidebar, we haven't been prefixing with `Fundbox-` or `Fbx-`
```suggestion
const stories = storiesOf('Components/Logo', module)
```